dissect/target/plugins/os/unix/log/auth.py

```diff
@@ -0,0 +1,61 @@
+import re
+from itertools import chain
+from typing import Generator
+
+from flow.record.fieldtypes import path
+
+from dissect.target.helpers.fsutil import TargetPath
+from dissect.target.helpers.record import TargetRecordDescriptor
+from dissect.target.helpers.utils import year_rollover_helper
+from dissect.target.plugin import Plugin, export
+
+AuthLogRecord = TargetRecordDescriptor(
+    "linux/log/auth",
+    [
+        ("datetime", "ts"),
+        ("string", "message"),
+        ("string", "source"),
+    ],
+)
+
+ts_regex = r"^[A-Za-z]{3}\s*[0-9]{1,2}\s[0-9]{1,2}:[0-9]{2}:[0-9]{2}"
+RE_TS = re.compile(ts_regex)
+RE_TS_AND_HOSTNAME = re.compile(ts_regex + r"\s\w+\s")
```

Comment:
```suggestion
_TS_REGEX = r"^[A-Za-z]{3}\s*[0-9]{1,2}\s[0-9]{1,2}:[0-9]{2}:[0-9]{2}"
RE_TS = re.compile(_TS_REGEX)
RE_TS_AND_HOSTNAME = re.compile(_TS_REGEX + r"\s\w+\s")
```
To stay in the spirit of uppercase constants.